string name
string clan
string address
string designation
int64 age
float64 reading1
float64 reading2
float64 reading3